node/cmd/guardiand/adminserver.go

```diff
@@ -874,6 +877,9 @@ func (s *nodePrivilegedService) DumpRPCs(ctx context.Context, req *nodev1.DumpRP
 	rpcMap["polygonRPC"] = *polygonRPC
 	rpcMap["pythnetRPC"] = *pythnetRPC
 	rpcMap["pythnetWS"] = *pythnetWS
+	if s.testnetMode {
+		rpcMap["sepoliaRPC"] = *sepoliaRPC
+	}
```

Comment:
No, Goerli is just like any other testnet, so if you are running in testnet, the ethRPC would be the goerli one. Since Sepolia is handled separately (so we can run both goerli and Sepolia), it shows up here as a separate RPC, which we don't want to include in the mainnet output.